StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -70,6 +70,9 @@
 /* Text providing link to terms for ACH payments */
 "By continuing, you agree to authorize payments pursuant to <terms>these terms</terms>." = "By continuing, you agree to authorize payments pursuant to <terms>these terms</terms>.";
 
+/* Mandate text displayed when paying via Link instant debit. */
+"By continuing, you agree to authorize payments pursuant to these <terms>terms</terms>." = "By continuing, you agree to authorize payments pursuant to these <terms>terms</terms>.";
+
```

Comment:
probably outside the scope of this PR, but for ACH, it's:
```
<terms>these terms</terms>.
```
and for this one it is:
```
these <terms>terms</terms>.
```